kornia/feature/lightglue.py

```diff
@@ -424,6 +428,9 @@ def __init__(self, features: str = "superpoint", **conf_) -> None:  # type: igno
             if features in ["keynet_affnet_hardnet"]:
                 fname = "keynet_affnet_hardnet_lightlue.pth"
                 url = "http://cmp.felk.cvut.cz/~mishkdmy/models/keynet_affnet_hardnet_lightlue.pth"
+            elif features in ["dedodeb"]:
+                fname = "dedodeb_lightglue.pth"
+                url = "http://cmp.felk.cvut.cz/~mishkdmy/models/dedodeb_lightglue.pth"
```

Comment:
should we move these weights to hf?

Reply:
would be great to have a backup yes. Not sure how much accessibility @ducha-aiki has on that server

Reply:
That is my personal space on university website. 
However, if you want to have backup on HF - sure, go for it :)